src/util/zip.ts

```diff
@@ -18,14 +18,24 @@ import JSZip from 'jszip'
 export const create = async (
   zipPath: string,
   files: string[],
-  options: { remove: boolean }
+  options: { remove: boolean; dir: string }
```

Comment:
Using a `;` here looks weird to me, I would have though you use a `,`. But prettier changed it to a `;`. Maybe I'm just coming from Python land.

Reply:
It's a semicolon because the snippet in question is a TypeScript interface and not a JavaScript object 🙃

Reply:
Yeah, of course. the `;` is instead of a newline.